recipes/ESMValTool-sample-data/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set python_min = "3.10" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: "esmvaltool-sample-data"
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/e/ESMValTool_sample_data/esmvaltool_sample_data-{{ version }}.tar.gz
+  sha256: ce8f18cd48e70db5b7e54d7b0d549b8f59a668f5a322b048eb60ee4bd0dc8d82
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip !=21.3
+    - setuptools >=40.6.0
+    - setuptools_scm >=6.2
+  run:
+    - python >={{ python_min }}
+    - esgf-pyclient
+    - iris >=3.6
+    - myproxyclient
+
+test:
+  requires:
+    - python {{ python_min }}
+  imports:
+    - esmvaltool_sample_data
+
+about:
+  home: https://github.com/ESMValGroup/ESMValTool_sample_data
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: 'PyActiveStorage: provides a Python client to local or remote active storage (aka computational storage).'
```

Comment:
Is this summary correct?

Reply:
done in 1ef9dd8b74b5b78bfe0cec459e1321d76dde42fb :beer: